src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+let populationsList = document.getElementsByClassName('population');
+const total = document.querySelector('.total-population');
+const average = document.querySelector('.average-population');
+
+populationsList = [...populationsList]
+  .map(value => +value.textContent.split(',').join(''));
```

Comment:
it may lead to unexpected issues, so just create another var here

Reply:
fixed